packages/diracx-web-components/src/components/shared/Error.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Alert, AlertTitle, Button } from "@mui/material";
+import ReplayIcon from "@mui/icons-material/Replay";
+
+/**
+ * This component is used to display an error message
+ *
+ * @param s The error message (optional)
+ * @param reset The function to call to reset the error (optional)
+ * @returns Error component
+ */
+export function ErrorComponent({
```

Comment:
I wonder if `Component` should be part of the name as it's a redundant information.
What about `ErrorBox`? Feel free to suggest other names if you have a better idea.

```suggestion
export function ErrorBox({
```